server/engine/MapCard.js

```diff
@@ -0,0 +1,19 @@
+const WordRoles = require("./WordRoles");
+
+const blueAgents = new Array(9).fill(WordRoles.BLUE);
```

Comment:
i'm guessing this may just be work in progress because blue and red agents could flip between 9 or 8 depending on the map (randomized)

Reply:
Thanks, I'll fix this.